docs/migrations.md

```
@@ -0,0 +1,358 @@
+
+**Who needs this migration:**
+
+- Only users upgrading from a version that used the `holds_detected` JSON column
+- New installations will not have this column and do not need this migration
+
+### Prerequisites
+
+Before running this migration, ensure:
+
+1. **You have a database backup**
+2. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)